entities/src/login.rs

```diff
@@ -9,8 +8,7 @@ use serde::{Deserialize, Serialize};
 pub struct Model {
     #[sea_orm(primary_key)]
     pub id: u64,
-    #[sea_orm(unique)]
-    #[serde(serialize_with = "serialization::uuid_as_ulid")]
```

Comment:
we don't need the serialize decorator anymore?

Reply:
we do. just missed it being replaced